orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/PeeringAgent.kt

```diff
@@ -189,6 +192,24 @@ class PeeringAgent(
     }
   }
 
+  /**
+   * If we have any custom peerers, invoke them
+   */
+  private fun invokeCustomPeerers() {
+    customPeerers.forEach { customPeerer ->
+      val peererName = customPeerer.javaClass.simpleName
+
+      try {
+        log.info("Starting peering with custom peerer '$peererName'")
+        customPeerer.doPeer(srcDB, destDB, peeredId)
```

Comment:
would be neat to wrap this call in a `Timer::record` call

Reply:
yeah, i was going back and forth... at the end settled that the implementer should do that so they know what metric to track... but i can be convinced otherwise... marinading

Reply:
I really feel strongly that it should be done here to enforce that it is actually done so that there is a consistent metric naming/tagging scheme. Plus if you're implementing the custom peerer the timing comes for free out of the box, which is a nice touch.